apis/vshn/v1/vshn_nextcloud.go

```diff
@@ -188,6 +188,19 @@ type CollaboraSpec struct {
 	FQDN string `json:"fqdn,omitempty"`
 }
 
+type addon interface {
```

Comment:
You forgot to cleanup after our short session about addons.

Reply:
ah, shit :D